src/main/index.js

```diff
@@ -997,7 +997,23 @@ function runApp() {
           { role: 'zoomout' },
           { role: 'zoomout', accelerator: 'CmdOrCtrl+numsub', visible: false },
           { type: 'separator' },
-          { role: 'togglefullscreen' }
+          { role: 'togglefullscreen' },
```

Comment:
Allowing last comma on multiline makes duplicating last code and modifying it much easier, also no more diff on N-1th line